### PR TITLE
PP-7583 URL Restructure - API Keys

### PR DESCRIPTION
**Pull request by iqbalgds**
with @sandorarpa

- paths.js
  - Move API paths to `account` section.
- routes.js
  - Add all API Key paths to the `account router`
- Update all API key views and controllers
  - Update all references to API key pages to use the new URL structure
    - i.e. /account/<gateway-account>/api-keys/*
- Update functional tests to use mock calls for the new account middleware
- Updated nav-builder to use new account URLs for the API Keys
- Updated template test for the navigation

### CODE REVIEW

File: app/utils/nav-builder.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ const serviceNavigationItems = (currentPath, permissions, type) => {
   navigationItems.push({
     id: 'navigation-menu-settings',
     name: 'Settings',
-    url: type === 'card' ? paths.settings.index : paths.apiKeys.index,
+    url: paths.settings.index,
```
Comment (iqbalgds) [inline app/utils/nav-builder.js:52]: Removed the DD code here - now only handles the `card` payments.

Comment (stephencdaly) [reply app/utils/nav-builder.js:52]: It won't allow me to comment on it, but need to update line 56: `paths.apiKeys` -> `paths.account.apiKeys` to make sure the "Settings" tab is highlighted on the top nav bar when you're on this page.

**Review by stephencdaly — CHANGES_REQUESTED** [review]
Looks good, just one very small thing that needs updating

**Comment by iqbalgds** [comment]
Good spot - just updated it

**Review by stephencdaly — APPROVED** [review]

**Pull request merged**